autoload/pandoc/folding.vim

```diff
@@ -109,9 +109,12 @@ endfunction
 function! pandoc#folding#FoldExpr()
     " with multiple splits in the same buffer, the folding code can be called
     " way too many times too often, so it's best to disable it to keep good
-    " performance.
-    if count(map(range(1, winnr('$')), 'bufname(winbufnr(v:val))'), bufname("")) > 1
-        return
+    " performance. Only enable when using the built-in method of improving
+    " performance of folds.
+    if g:pandoc#folding#fastfolds == 1
```

Comment:
One can think of checking for `if g:pandoc#... == 1` instead of only `if g:pandoc#...`  everywhere, especially where the documentation only cites `0` and `1` as possible values; otherwise values `> 1` might cause unexpected behavior.

Reply:
That's a good idea. I remember in some cases it was intentional, though (actually I considered doing something like that here, because it might have made sense to separate the autocommand setup from the multiple splits workaround, so either different integer values or a flag could have worked.)